lib/hardware_services/inventory_drivers/hil.yml

```diff
@@ -0,0 +1,2 @@
+url: http://127.0.0.1:5000    # the address of HIL server
```

Comment:
Yes. We discussed that. And I agreed with you. But I don't see it added.

So can we merge this first and then we can move this line anywhere?